cryptofeed/exchange/deribit.py

```diff
@@ -153,24 +154,34 @@ async def _ticker(self, msg: dict, timestamp: float):
             'bid_iv': Decimal(m.get('bid_iv', 0)),
             'ask_iv': Decimal(m.get('ask_iv', 0)),
             'mark_iv': Decimal(m.get('mark_iv', 0)),
+            'oi': m.get('open_interest'),   # in quote ccy
             **m.get('greeks', {}),  # vega, theta, rho, gamma, delta
         }
         await self.callback(TICKER, feed=self.id,
                             symbol=pair,
-                            bid=Decimal(msg["params"]["data"]['best_bid_price']),
-                            ask=Decimal(msg["params"]["data"]['best_ask_price']),
+                            bid=Decimal(m['best_bid_price']),
+                            ask=Decimal(m['best_ask_price']),
                             timestamp=ts,
                             receipt_timestamp=timestamp,
                             **extra_fields)
 
-        if "current_funding" in msg["params"]["data"] and "funding_8h" in msg["params"]["data"]:
-            await self.callback(FUNDING, feed=self.id,
-                                symbol=pair,
-                                timestamp=ts,
-                                receipt_timestamp=timestamp,
-                                rate=msg["params"]["data"]["current_funding"],
-                                rate_8h=msg["params"]["data"]["funding_8h"])
-        oi = msg['params']['data']['open_interest']
+        if "current_funding" in m and "funding_8h" in m:
```

Comment:
Is this because only certain messages in the same pair will include the funding info? Or why do we need to cache the latest message?

Reply:
yeah, not all pairs have the funding info (only perps)